.github/workflows/docs-generate.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - name: Check Generated Docs Differences
         run: |
-          DOCS_DIFF=$(git ls-files --others --modified --exclude-standard api)
+          DOCS_DIFF=$(git ls-files --others --modified --exclude-standard)
```

Comment:
i dont get how this change satisfies my comment - https://github.com/unitycatalog/unitycatalog/pull/1231#discussion_r2551279566
can you explain a bit more.